Makefile

```diff
@@ -179,10 +179,12 @@ tmp/weave-scope.tgz: $(shell find client/app -type f) $(SCOPE_UI_BUILD_UPTODATE)
 else
 
 client/build/index.html:
-	cd client && npm run build
+	test "true" = "$(SCOPE_SKIP_UI_ASSETS)" && { mkdir -p client/build; } || \
```

Comment:
You are right, there is no difference in what is executed, I removed them. In the subsequent line I think keeping it makes sense for improved readability.